consensus/entry_point_selector/entry_point_selector.h

```diff
@@ -27,14 +27,42 @@ typedef struct entry_point_selector_s {
   bool testnet;
 } entry_point_selector_t;
 
+/**
+ * Initializes an entry point selector
+ *
+ * @param eps The entry point selector
+ * @param mt A milestone tracker
+ * @param tangle A tangle
+ * @param tester True if testnet, false otherwise
+ *
+ * @return a status code
+ */
 extern retcode_t iota_consensus_entry_point_selector_init(
     entry_point_selector_t *const eps, milestone_tracker_t *const mt,
     tangle_t *const tangle, bool const testnet);
 
-extern retcode_t iota_consensus_get_entry_point(
+/**
+ * Gets an entry point for the random walk.
+ * Given a depth N, returns a N-deep milestone, i.e. (latest_solid - depth)th
+ * milestone.
+ *
+ * @param eps The entry point selector
+ * @param depth The depth
+ * @param ep The entry point
+ *
+ * @return a status code
+ */
+extern retcode_t iota_consensus_entry_point_selector_get_entry_point(
```

Comment:
Why the explicit extern? Because there can be multiple impls?

Reply:
No specific reason, I saw it many times elsewhere and somehow thought it was part of the coding style, I'll remove.

Reply:
The motivation for that is link/build-time pluggable implementations. Might make sense here in the future as well, we'll see.